src/cloud-api-adaptor/podvm/files/etc/systemd/system/confidential-data-hub.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Confidential Data Hub TTRPC API Server
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/local/bin/confidential-data-hub -c /run/confidential-containers/cdh.toml
```

Comment:
Does this mean we can remove `Environment=CDH_CONFIG_PATH=/run/confidential-containers/cdh.toml` from the kata-agent.service (and potentially the code that reads it in the guest-components)?